code/modules/unit_tests/screenshot_humanoids.dm

```diff
@@ -17,7 +17,7 @@
 	var/mob/living/carbon/human/dwarf = allocate(/mob/living/carbon/human/dummy/consistent)
 	dwarf.set_species(/datum/species/dwarf)
 	dwarf.facial_hairstyle = "Beard (Dwarf)"
-	dwarf.update_hair(is_creating = TRUE)
+//	dwarf.update_hair(is_creating = TRUE) //don't know what you was doing with this. striked for now. bad arguement.
```

Comment:
if there's something wrong with this dorf code make it part of a separate PR, sorry